docs/tutorials/00_amplitude_estimation.ipynb

```diff
@@ -0,0 +1,582 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Amplitude Estimation\n",
+    "\n",
+    "Provided with an operator $\\mathcal{A}$ that acts as\n",
```

Comment:
```suggestion
    "Given an operator $\\mathcal{A}$ that acts as\n",
```
`Provided with` seems a little odd there to me - `Given` some criterion seems more the way these things are usually stated